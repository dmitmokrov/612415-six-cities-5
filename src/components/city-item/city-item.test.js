import React from 'react';
import renderer from 'react-test-renderer';
import CityItem from './city-item';

const noop = () => {};
const city = `London`;

describe(`Should CityItem render correctly`, () => {
  it(`Active`, () => {
    const tree = renderer
      .create(<CityItem
        city={city}
        isActive={true}
        changeCity={noop}
      />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

  it(`Not active`, () => {
    const tree = renderer
      .create(<CityItem
        city={city}
        isActive={false}
        changeCity={noop}
      />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
