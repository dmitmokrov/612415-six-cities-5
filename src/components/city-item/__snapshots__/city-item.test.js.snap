// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should CityItem render correctly Active 1`] = `
<li
  className="locations__item"
>
  <a
    className="locations__item-link tabs__item tabs__item--active"
    href="#"
    onClick={[Function]}
  >
    <span>
      London
    </span>
  </a>
</li>
`;

exports[`Should CityItem render correctly Not active 1`] = `
<li
  className="locations__item"
>
  <a
    className="locations__item-link tabs__item "
    href="#"
    onClick={[Function]}
  >
    <span>
      London
    </span>
  </a>
</li>
`;
